git-active-branches

```diff
@@ -1,9 +1,40 @@
 #!/bin/sh
 set -eu
 
-# TODO: Make the max age configurable
+since='3.weeks.ago'
+
+usage() {
+    echo "usage: git active-branches [--since|--after <date>]" >&2
+    echo >&2
+    echo "Unless specified, <date> defaults to \"$since\". For examples see:" >&2
+    echo "https://git-scm.com/book/en/v2/Git-Basics-Viewing-the-Commit-History" >&2
+    exit 2
+}
+
+if [ $# -gt 0 ]; then
+    case $1 in
+        # 'git log' accepts either '--since' or '--after' as synonyms
+        --since|--after)
```

Comment:
Doing command line argument parsing manually is a very tricky process with lots of subtleties to handle. For consistency, I'd therefore prefer sticking to the `getopts` style arg parsing I'm using elsewhere in the project. This will unfortunately mean that we'll have to work with single-char flags, since I don't know of a portable option parser other than `getopts`. Perhaps we can use `-s` for since, and `-a` for after?

Reply:
`getopts` it is!

Do you even care that much about `--after`? Because I wouldn't have, except to support exactly the same options as `git log`. Now, if we are only going with the short options only, then we can probably forget about the `--after` entirely.

Reply:
You're right, I'm fine just supporting `-s`. I don't see a use case where I would ever want to use `-a` myself, so I'm fine not supporting this.